scalabel/eval/tagging.py

```diff
@@ -17,6 +32,434 @@
 from .utils import reorder_preds
 
 
+def _column_or_1d(y: NDArrayI32) -> NDArrayI32:
+    """Ravel column or 1d numpy array, else raises an error."""
+    y = np.asarray(y)
+    shape = np.shape(y)
+    if len(shape) == 1:
+        return np.ravel(y)
+    if len(shape) == 2 and shape[1] == 1:
+        return np.ravel(y)
+
+    raise ValueError(
+        "y should be a 1d array, got an array of shape {} instead.".format(
+            shape
+        )
+    )
+
+
+def _num_samples(x: NDArrayI32) -> int:
```

Comment:
This shouldn't be necessary, you can replace it with len(y) at where its being called